pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -240,6 +240,7 @@ type ProjectRoleTemplateBinding struct {
 	RoleTemplateName string `json:"roleTemplateName,omitempty" norman:"required,noupdate,type=reference[roleTemplate]"`
 
 	// ServiceAccount is the name of the service account bound as a subject. Immutable.
+	// Deprecated.
```

Comment:
I'd recommend using the `+kubebuilder:deprecatedversion:warning=string` annotation here so that we can use some controller-runtime tools to warn people about this field. More on that here: https://book.kubebuilder.io/reference/markers/crd.

If you choose not to elaborate on this field, elaborate why on thread.

Reply:
Discussed offline - I had not been able to find a special deprecation string for a field, only for a type, which is not what we want here.

Reply:
Resolving, there doesn't appear to be a better way to do this right now.